defaultsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "githubClientID": ["YOUR GITHUB CLIENT ID HERE"],
```

Comment:
I don't think you want the values for these variables to all be wrapped in arrays